packages/fxa-event-broker/lib/grant.ts

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
File naming nit: this doesn't have anything to do with "grants" the way the oauth-server means them, this file would probably be more clearly named as "jwts.ts" or "setokens.js" or similar; (naming it "set.js" would unfortunately be confusing for readers who don't know that's suppose to be an acronym...)

(The oauth-server could really use a glossary, but that's a separate issue...)